[inductor] Type annotate constant_folding.py

zhxchen17 opened the pull request:
Summary: Type annotate constant_folding.py

Test Plan: mypy

Reviewed By: angelayi

Differential Revision: D60063872


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

eellison reviewed (commented):
Looks good but still need to remove `# mypy: allow-untyped-defs`

zhxchen17 committed:
[inductor] Type annotate constant_folding.py (#131364)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/131364

Type annotate constant_folding.py

Test Plan: mypy

Reviewed By: angelayi

Differential Revision: D60063872
